docs/development/qubes_staging.rst

```diff
@@ -53,17 +53,17 @@ In ``dom0``:
 
 .. code:: sh
 
-   qvm-start sd-trusty-base --cdrom=sd-dev:/home/user/ubuntu-14.04.5-server-amd64.iso
+   qvm-start sd-trusty-base --cdrom=sd-dev:/home/user/ubuntu-16.04.5-server-amd64.iso
```

Comment:
sd-trusty-base -> sd-staging-base

I recommend we also specify ubuntu-16.04.6 (see #4234)

Reply:
Typo corrected. I'm gonna stick in another PR updating 16.04.5 refs throughout the docs, will fix it here with said PR for consistency's sake.

Reply:
Oh wait, this isn't on develop yet! I'll just do it in this PR.